pillar/availability_zone.sls

```diff
@@ -0,0 +1,3 @@
+availability_zone:
+  region: 'region1'
+  zone: 'zone1'
```

Comment:
I think these belong in `pillar/params.sls`, as we expect users to override them. Should we maybe default to "unknown" for both rather than potentially telling end-users we know something about the topology?

(Also - Total guess that all user settable pillars are in `params.sls`, It just looks like they are..